helm/Chart.yaml

```
@@ -5,6 +5,6 @@ type: application
 version: 0.1.0
 appVersion: "0.1.0"
 dependencies:
-  - name: cas-postgres
-    version: "0.8.3"
-    repository: https://bcgov.github.io/cas-postgres/
+  - name: spilo
+    version: 0.1.0
```

```suggestion
    version: 0.2.0
```
Don't forget to run `helm dep up` after updating to update the `Chart.lock`